overlord/configstate/configcore/prompting.go

```diff
@@ -0,0 +1,72 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+//go:build !nomanagers
+// +build !nomanagers
```

Comment:
Is the syntax with the space after the comment correct?

Reply:
I think a typo from mvo, I've removed in most recent commit.